specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2023-05-01-privatepreview/examples/AlertProcessingRules_Create_or_update_add_correlation.json

```diff
@@ -0,0 +1,231 @@
+{
+  "parameters": {
+    "api-version": "2023-05-01-privatepreview",
+    "subscriptionId": "subId1",
+    "resourceGroupName": "alertscorrelationrg",
+    "alertProcessingRuleName": "CorrelationExample",
+    "alertProcessingRule": {
+      "location": "Global",
+      "tags": {},
+      "properties": {
+        "scopes": [
+          "/subscriptions/subId1"
+        ],
+        "actions": [
+          {
+            "actionType": "Correlation",
+            "correlateBy": [
+              {
+                "field": "essentials.alertRuleName",
+                "type": "Simple"
```

Comment:
Is "type" an enum with only "Simple" and "Grouped" values, or should it be an array of strings?

Reply:
lets sync offline - not sure if we need both types in the initial version

Reply:
The question is breaking change. 
I was thinking we would want to use the type, like "actionType". i.e. as an "extensible enum" and a 'discriminator' for the different types of correlation. we used this patter to avoid breaking change when extending the API. (although we will only be testing this now :) )

Reply:
Btw- the names , Type and simple were contested. We may need to think about better names.

Reply:
Remove the "type"